src/js/components/agencyV2/overview/BarChart.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * BarChart.jsx
+ * Created by Max Kendall 4/13/21
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BarChart = ({
+    agencyBudgetByYear,
+    selectedFy
+}) => {
+    const renderBars = () => {
+        const highestYear = agencyBudgetByYear.reduce((acc, obj) => {
+            if (obj.budget > acc) return obj.budget;
+            return acc;
+        }, 0);
+        return agencyBudgetByYear
+            .filter((obj) => {
+                // show preceding 4 years only.
+                if (parseInt(selectedFy, 10) <= 2021 && obj.year <= 2021) return true;
+                if (parseInt(selectedFy, 10) > 2021 && obj.year >= parseInt(selectedFy, 10) - 4) return true;
+                return false;
+            })
+            .sort((a, b) => a.year - b.year)
+            .map(({ year: fy, budget: total }, i) => {
+                const fyStr = String(fy);
+                return (
+                    <li className="bar-chart__bar" key={i}>
+                        <span
+                            className={`${fyStr === selectedFy ? 'active-fy ' : ''}`}
+                            style={{
+                                height: `${(total / highestYear) * 100}%`,
```

Comment:
I am not sure if this is actually accurate? It looks like we are taking the `highestYear` which is really the max budget of all the possible fiscal years and then setting the height to be proportionate to all fiscal year budgets when these rectangle's heights should be proportionate to each other? So we should move `highestYear` to after we filter to four years we are going to show the user.

Reply:
yeah I might be missing something.

Reply:
OK. Confirmed. You're right. Updated.